### DISCUSSION TITLE
Can I purchase more storage?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#127271] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think I have reached a storage cap when I commit and deploy to my page. Is there a way I could pay monthly to upload more to my page? My workflow builds are failing with some kind of "Contentpage" error and "stack exceeded." I'm uploading markdown notes and pictures.

**Comment by gil-son** [comment]
Hello, how are you? 

Unfortunately, storage limit can be [ just 1 GB](https://docs.github.com/en/enterprise-server@3.9/pages/getting-started-with-github-pages/about-github-pages)

But you can check more possibilities on Github Support

Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Comment by yoannchaudet** [comment]
Hello,

GitHub Pages sites are [recommended to stay under 1GB](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits).

While you can purchase extra LFS storage for Git, you cannot do that for Pages.

If you have a link to the site in question, I can take a look at. your "stack exceeded" error. That sounds peculiar.

**Comment by vdevcode** [comment]
GitHub does have storage limits for repositories, which can cause issues when you exceed them. However, GitHub offers various solutions to manage and potentially increase your storage capacity.
GitHub Repository Size Limits

    Individual File Limit: No single file in a repository can be larger than 100 MB.
    Repository Size Limit: While there is no strict limit, repositories larger than 1 GB are discouraged, and you might face issues if you exceed this. A soft limit of 5 GB is often referenced.
    GitHub Pages: The limit for GitHub Pages sites is generally around 1 GB.

Upgrading Storage

    GitHub Pro: Individual developers can upgrade to GitHub Pro for $4/month, which includes additional features but does not necessarily increase repository storage limits significantly.
    GitHub Team/Enterprise: These plans are for organizations and come with enhanced features, support, and potentially more storage options. Pricing varies and typically starts at $4/user/month for Team and $21/user/month for Enterprise.
    
   and then Contacting GitHub Support

If these solutions do not resolve your issues, you might need to contact GitHub Support for further assistance. They can provide guidance tailored to your specific situation and might offer additional solutions based on your needs.
Summary

    Upgrade to a plan that suits your needs (GitHub Pro, Team, or Enterprise).
    Use Git LFS for large files.
    Split your repository if it's too large.
    Remove unnecessary files or use external storage solutions.
    Optimize your build process to avoid stack exceeded errors.

By following these steps, you should be able to manage your repository's size and avoid hitting storage caps, ensuring smooth deployment to GitHub Pages.
